visualization/app/codeCharta/ui/attributeSideBar/attributeSideBarHeaderSection/nodePath/fileCountDescription.selector.ts

```diff
@@ -0,0 +1,15 @@
+import { createSelector } from "../../../../state/angular-redux/createSelector"
+import { selectedNodeSelector } from "../../../../state/selectors/selectedNode.selector"
+
+type Node2FileCountDescription = { attributes?: { unary?: number } }
+
+export const node2fileCountDescription = (node: Node2FileCountDescription | undefined) => {
+	if (!node) return
```

Comment:
May this selector really be called with an undefined node?

Reply:
yes. This node comes from selected building, which can be undefined, if no building is selected in the map.

Reply:
I understood that. I just wonder if the functionality itself is ever called in case no building is selected? I would expect this to trigger on building selection?

Reply:
same as [above](https://github.com/MaibornWolff/codecharta/pull/2527#discussion_r757949475)